src/Kmd.Logic.FileSecurity.Client/ModelMarshaller.cs

```diff
@@ -0,0 +1,81 @@
+﻿using Kmd.Logic.FileSecurity.Client.Models;
+using Kmd.Logic.FileSecurity.Client.ServiceMessages;
+using Kmd.Logic.FileSecurity.Client.Types;
+using Microsoft.Rest;
+using System.Threading.Tasks;
+
+namespace Kmd.Logic.FileSecurity.Client
+{
+    internal static class ModelMarshaller
```

Comment:
As for now I don't see any value of having that. In docgen module there was a reason to have that, here not really. Let's use models from generated client directly

Reply:
Removed the class.